src/lib/github.js

```diff
@@ -81,7 +82,7 @@ module.exports = async function (pkg, info) {
     type: 'input',
     name: 'username',
     message: 'What is your GitHub username?',
-    default: npm.config.get('username'),
+    default: (parse.sync({path: gitConfigPath}).github) ? parse.sync({path: gitConfigPath}).github.user : npm.config.username,
```

Comment:
I don’t have that configuration in my `~/.gitconfig` file, how do I add it? Maybe you could  add a section like you did for the npm username? That’d be 💯

Reply:
Done!